app/tests/integration/nimbus/conftest.py

```diff
@@ -54,32 +75,39 @@ def _verify_url(request, base_url):
         session.get(base_url, verify=False)
 
 
-@pytest.fixture(name="timeout_length", scope="session")
-def fixture_timeout_length():
-    return 60
+@pytest.fixture
+def kinto_url():
+    return "http://kinto:8888/v1/admin/"
+
+
+@pytest.fixture
+def kinto_review_url(kinto_url, default_data):
+    return urljoin(kinto_url, APPLICATION_KINTO_REVIEW_PATH[default_data.application])
+
+
+@pytest.fixture
+def experiment_url(base_url, default_data):
+    return urljoin(base_url, slugify(default_data.public_name))
 
 
 @pytest.fixture(
-    params=[
-        (BaseExperimentApplications.DESKTOP, "No Feature Firefox Desktop"),
-        (BaseExperimentApplications.FENIX, "No Feature Fenix"),
-        (BaseExperimentApplications.IOS, "No Feature iOS"),
-        (BaseExperimentApplications.KLAR, "No Feature Klar for Android"),
-        (BaseExperimentApplications.FOCUS, "No Feature Focus for Android"),
-    ],
-    ids=["DESKTOP", "FENIX", "IOS", "KLAR", "FOCUS"],
+    params=list(BaseExperimentApplications),
+    ids=[application.name for application in BaseExperimentApplications],
```

Comment:
I like this but I don't at the same time

Reply:
Yeah we could leave it as it is, I did this in a bunch of other places in the rest of the Nimbus project just so the list of applications is written in one place instead of in a bunch places.

Reply:
Can we add a comment just to help with readability?

Reply:
Shore 👍